### PR TITLE
Override ckConfigurator for local needs: allow plugin qtiUnderline

### PR DESCRIPTION
**Pull request by jsconan**
depends on : https://github.com/oat-sa/tao-core/pull/329

### CODE REVIEW

File: views/js/controller/creator/helpers/ckConfigurator.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```
Comment (krampstudio) [inline views/js/controller/creator/helpers/ckConfigurator.js]: Please change the header

File: views/js/controller/creator/helpers/ckConfigurator.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     * options.dtdOverrides         -> @see dtdOverrides which pre-defines them
+     * options.positionedPlugins    -> @see ckConfig.positionedPlugins
+     * options.qtiImage             -> enables the qtiImage plugin
+     * options.qtiInclude           -> enables the qtiInclude plugin
+     * options.underline            -> enables the underline plugin
+     * options.mathJax              -> enables the mathJax plugin
+     *
+     * @param editor instance of ckeditor
+     * @param toolbarType block | inline | flow | qtiBlock | qtiInline | qtiFlow | reset to get back to normal
+     * @param options is based on the CKEDITOR config object with some additional sugar
```
Comment (krampstudio) [inline views/js/controller/creator/helpers/ckConfigurator.js]: Better way to document options: 

```javascript
/**
 *  @param {Object} [options] -   is based on the CKEDITOR config object with some additional sugar
 *  @param {Boolean} [options.qtiImage] - enables the qtiImage plugin
 * ...
 */
```

**Comment by krampstudio** [comment]
Check also for missing `use strict` and wrong header everywhere

**Comment by jsconan** [comment]
I added missing 'use strict'; and fixed wrong headers

2015-05-21 11:45 GMT+02:00 Bertrand CHEVRIER <notifications@github.com>:

> Check also for missing use strict and wrong header everywhere
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oat-sa/extension-tao-testqti/pull/59#issuecomment-104205180>
> .
>



-- 




*Jean-Sébastien CONAN*
Software Engineer

•   Email:   jean-sebastien.conan@vesperiagroup.com
•   GSM: (+352) 621 320 343

*Vesperia Group*
148, Route d'Arlon
L-8010 Strassen
Luxembourg

http://www.vesperiagroup.com/

**Pull request merged**